airbyte-server/src/main/java/io/airbyte/server/handlers/WebBackendConnectionsHandler.java

```diff
@@ -113,6 +114,9 @@ public WebBackendConnectionReadList webBackendListConnectionsForWorkspace(final
     final Map<UUID, JobRead> runningJobByConnectionId =
         getRunningJobByConnectionId(standardSyncs.stream().map(StandardSync::getConnectionId).toList());
 
+    final Map<UUID, ActorCatalogFetchEvent> newestFetchEventsByActorId =
+        configRepository.getMostRecentActorCatalogFetchEventForSources(new ArrayList<>());
```

Comment:
@benmoriceau why are we adding more db queries into this handler? it's really specifically _not_ supposed to be making direct database calls.

Reply:
@cgardens It was not very clear that the repository shouldn't have been used here. Especially since it was recently in in the same endpoint and other endpoints to get data from the DB. The ticket related to this PR https://github.com/airbytehq/airbyte/issues/17526 is only needing the described functionality in the webBackend endpoint only so it made sense to add that direct call in the WebBackendHandler.

Reply:
https://github.com/airbytehq/airbyte/blob/master/airbyte-server/src/main/java/io/airbyte/server/handlers/WebBackendConnectionsHandler.java#L83

What else do we need to add to make it clear?

Reply:
This comment is easy to miss when the same field is being used within the same function [here](https://github.com/airbytehq/airbyte/blob/master/airbyte-server/src/main/java/io/airbyte/server/handlers/WebBackendConnectionsHandler.java#L156). When you are looking at the function to update, it is not explicit that this is a deprecated field. Something like [this](https://github.com/airbytehq/airbyte/pull/19692) would make it very explicit that this is deprecated.